webroot/index.php

```diff
@@ -33,10 +34,14 @@
 
 function do_cleanup()
 {
-	global $dbh;
+	global $dbh, $config;
 	
-	$stmt = $dbh->prepare("DELETE FROM `pastes` WHERE `expires` < NOW()");
-	$stmt->execute();
+	if($config['daba_type'] == "mysql") {
```

Comment:
The way to implement multiple databases is not by branching at every place where a query takes place. We'll need to add a database interface in between the function handlers and the database: the current approach will make functions unreadable, especially when someone for instance adds Oracle support.